gsa/src/web/pages/audits/__tests__/actions.js

```diff
@@ -229,6 +229,91 @@ describe('Audit Actions tests', () => {
     );
   });
 
+  test('should not call click handlers for stopped audit without permissions', () => {
+    const audit = Audit.fromElement({
+      status: AUDIT_STATUS.stopped,
+      alterable: '0',
+      last_report: {report: {_id: 'id'}},
+      permissions: {permission: [{name: 'get_task'}]},
+      target: {_id: 'id', name: 'target'},
+      usage_type: 'audit',
+    });
+
+    const handleAuditClone = jest.fn();
+    const handleAuditDelete = jest.fn();
+    const handleAuditDownload = jest.fn();
+    const handleAuditEdit = jest.fn();
+    const handleAuditResume = jest.fn();
+    const handleAuditStart = jest.fn();
+    const handleAuditStop = jest.fn();
+    const handleReportDownload = jest.fn();
+
+    const {render} = rendererWith({capabilities: wrongCaps});
+    const {getAllByTestId} = render(
+      <Actions
+        entity={audit}
+        gcrFormatDefined={true}
+        links={true}
+        onAuditCloneClick={handleAuditClone}
+        onAuditDeleteClick={handleAuditDelete}
+        onAuditDownloadClick={handleAuditDownload}
+        onAuditEditClick={handleAuditEdit}
+        onAuditResumeClick={handleAuditResume}
+        onAuditStartClick={handleAuditStart}
+        onAuditStopClick={handleAuditStop}
+        onReportDownloadClick={handleReportDownload}
+      />,
+    );
+
+    const icons = getAllByTestId('svg-icon');
+
+    fireEvent.click(icons[0]);
+    expect(handleAuditStart).not.toHaveBeenCalledWith(audit);
+    expect(icons[0]).toHaveAttribute(
+      'title',
+      'Permission to start audit denied',
+    );
+
+    fireEvent.click(icons[1]);
+    expect(handleAuditResume).not.toHaveBeenCalledWith(audit);
+    expect(icons[1]).toHaveAttribute(
+      'title',
+      'Permission to resume audit denied',
+    );
+
+    fireEvent.click(icons[2]);
+    expect(handleAuditDelete).not.toHaveBeenCalledWith(audit);
+    expect(icons[2]).toHaveAttribute(
+      'title',
+      'Permission to move Audit to trashcan denied',
```

Comment:
Maybe adjust the upper case Audit for these icons as well? I know, I pledged for doing it step by step, but I didn't necessarily mean inconsistencies within audit icons (just across entities maybe).

Reply:
Inconsistencies across entities are not possible because delete, edit etc are always the entity icons that we use for every entity. That was precisely the reason I asked because if we change one type of icon we have to change all of them to avoid inconsistencies on the same page. But it might be better to do that in a separate PR.

Reply:
Ok, that's fine with me :)